manageiq-operator/pkg/helpers/miq-components/kafka.go

```diff
@@ -336,6 +336,9 @@ func ZookeeperDeployment(cr *miqv1alpha1.ManageIQ, scheme *runtime.Scheme) (*app
 		deployment.Spec.Strategy = appsv1.DeploymentStrategy{
 			Type: "Recreate",
 		}
+		if len(cr.Spec.AppAnnotations) > 0 {
```

Comment:
This is going to overwrite the backup annotations on https://github.com/ManageIQ/manageiq-pods/pull/677/files#diff-e23faf1d310cc4c5d8c16370e2cf9c9544e328f6dd349865dcbf585914565d51R331

Reply:
Good Catch 👍. I was only looking at the openshift.io/scc annotation which was there regardless so I thought I wasn't overwriting.    I'll update the PR to do a merge instead.  Thanks!!

Reply:
Ok, fix in this commit https://github.com/ManageIQ/manageiq-pods/pull/677/commits/ece18bd5433e970f98e8877a58f081886c16095c. I followed the pattern of the other methods, add*, and generalized it for Annotations. So it can also be used for other cases.   Thanks for catching this.